### DISCUSSION TITLE
Does this package support App Bridge 3.0

### DISCUSSION THREAD

**Discussion by ecomzier** [osiset/laravel-shopify#1219] [Q&A]
Does this package support App Bridge 3.0

**Comment by litalboa** [comment]
+1

**Reply by zeeshanhaider6236dev** [reply]
+1

**Comment by Kyon147** [comment]
It depends what you are trying to use with 3.0? The package just uses app bridge in a minimal form to load into the iframe and getting JWL tokens.

However with the push to using frontend tech like React or Vuejs - it might be more of a decision of the dev in the near future as @osiset and I recently have been talking about blade not being a solid option with the requirements needed by Shopify to have an app approved.

I don't see any reason by you can't use 3.0 but it depends largely on your specific setup and requirements.

**Comment by zeeshanhaider6236dev** [comment]
+1

**Comment by gnikyt** [comment]
Yes, were in a weird spot where Shopify is essentially forcing its toolset, meanwhile, we're trying to be flexible to allow Vue, React, or Blade for example, but it is increasingly becoming harder to support due to Shopify.
